src/pages/login.svelte

```diff
@@ -576,7 +604,7 @@
             <ul>
                 <li><ButtonLogin asLink>Sign In</ButtonLogin></li>
                 <li>
-                    <a href="https://create.anchor.link/" target="_blank"> Create new account</a>
```

Comment:
And here, also why is this using a href=#?

Reply:
I was getting a svelte warning without it there

Reply:
Hm, better to keep the create.anchor.link url and cancel the event instead then

Reply:
makes sense 👍